codecov.yml

```diff
@@ -6,3 +6,4 @@ ignore:
     - "./pkg/test/mocks/*.go"
     - "./swagger/*.go"
     - "./pkg/test/test_http_server.go"
+    - "./pkg/meta/proto_go/*.go"
```

Comment:
Is the path correct? I coudn't find any proto_go. Did you mean pkg/meta/proto ?

Reply:
Ah, this changed as well with my last update, it should be "pkg/meta/proto/gen"